base_projects/Drivetrain/src/main/cpp/Robot.cpp

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "Robot.h"
+
+#include <rev/CANSparkMax.h>
+
+#include <cstddef>
+#include <iostream>
+#include <sstream>
+
+#include <frc/ADXRS450_Gyro.h>
+#include <frc/AnalogGyro.h>
+#include <frc/Filesystem.h>
+#include <frc/RobotController.h>
+#include <frc/Spark.h>
+#include <frc/TimedRobot.h>
+#include <frc/romi/RomiGyro.h>
+#include <frc/smartdashboard/SmartDashboard.h>
+#include <units/angular_velocity.h>
+#include <wpi/FileSystem.h>
+#include <wpi/SmallString.h>
+#include <wpi/StringMap.h>
+
+// #include "AHRS.h"
+#include "rev/CANEncoder.h"
+
+Robot::Robot() : frc::TimedRobot(5_ms) {
+  std::error_code ec;
+
+  wpi::SmallString<128> path;
+  wpi::raw_svector_ostream os{path};
+
+  if constexpr (RobotBase::IsSimulation()) {
+    os << PROJECT_ROOT_DIR << "/src/main/deploy/config.json";
+
+  } else {
+    frc::filesystem::GetDeployDirectory(path);
+    os << "/config.json";
+  }
+
+  wpi::raw_fd_istream is{path.c_str(), ec};
+
+  if (ec) {
+    std::cout << "File error: " << path.c_str() << std::endl;
```

Comment:
This applies to other instances of std::cout.
```suggestion
    wpi::outs() << "File error: " << path.c_str() << '\n';
    wpi::outs().flush();
```